server/game/cards/03-WC/SaurianPlan.js

```diff
@@ -0,0 +1,16 @@
+const Card = require('../../Card.js');
+
+class SaurianPlan extends Card {
```

Comment:
Is this meant to be plant? Please rename the file too if so

Reply:
Fixed